torch/csrc/distributed/c10d/TCPStoreBackend.cpp

```diff
@@ -1,20 +1,10 @@
 
 #include <c10/util/irange.h>
-#include <fcntl.h>
 #include <algorithm>
 #include <array>
-#include <system_error>
 #include <unordered_map>
 #include <utility>
 
-#ifdef _WIN32
```

Comment:
These are safe to remove?

Reply:
Duplicated defined in #include <torch/csrc/distributed/c10d/Utils.hpp>